test/views/network_status_view_test.js

```diff
@@ -14,6 +14,7 @@ describe('NetworkStatusView', function() {
 
         after(function() {
             window.getSocketStatus = oldGetSocketStatus;
+            window.getSocketStatus = function() { return WebSocket.OPEN; };
```

Comment:
Do we mean to have the second statement overwrite the first?

Reply:
Yup. Adding a comment.